config/packages/security.yaml

```diff
@@ -17,6 +17,10 @@ security:
             anonymous: true
             stateless: false
 
+            two_factor:
+                auth_form_path: 2fa_login    # The route name you have used in the routes.yaml
```

Comment:
comments can be removed

Reply:
Fixed in 9944d94d01421fd5ea527806a2d3654727c45af8